test/unit/lib/cli/triage/fixtures/@serverless/compose/ts/project/serverless-compose.ts

```diff
@@ -1,8 +1,8 @@
 module.exports = {
-  "name": "serverless-compose-example",
```

Comment:
Let's not introduce changes to this file

Reply:
I don't see any changes to this file in my working copy

Reply:
There are clearly changes submitted here :) see the "Files changed" section